site/documentation/content/admin-guide/coap-adapter-config.md

```diff
@@ -28,8 +28,11 @@ The following table provides an overview of the configuration variables and corr
 | `HONO_COAP_BIND_ADDRESS`<br>`--hono.coap.bindAddress` | no | `127.0.0.1` | The IP address of the network interface that the secure port should be bound to.<br>See [Port Configuration]({{< relref "#port-configuration" >}}) below for details. |
 | `HONO_COAP_CERT_PATH`<br>`--hono.coap.certPath` | no | - | The absolute path to the PEM file containing the certificate that the protocol adapter should use for authenticating to clients. This option must be used in conjunction with `HONO_COAP_KEY_PATH`.<br>Alternatively, the `HONO_COAP_KEY_STORE_PATH` option can be used to configure a key store containing both the key as well as the certificate. |
 | `HONO_COAP_COAP_THREADS`<br>`--hono.coap.coapThreads` | no | 2 | The number of threads to use for processing CoAP message exchanges at the protocol layer. |
-| `HONO_COAP_CONNECTOR_THREADS`<br>`--hono.coap.connectorThreads` | no | 1 | The number of threads to use for receiving/sending UDP packets. The connector will start the given number of threads for each direction, outbound (sending) as well as inbound (receiving). |
+| `HONO_COAP_CONNECTOR_THREADS`<br>`--hono.coap.connectorThreads` | no | 2 | The number of threads to use for receiving/sending UDP packets. The connector will start the given number of threads for each direction, outbound (sending) as well as inbound (receiving). |
+| `HONO_COAP_DTLS_THREADS`<br>`--hono.coap.dtlsThreads` | no | 32 | The number of threads to use for processing DTLS message exchanges at the connection layer. |
+| `HONO_COAP_DTLS_RETRANSMISSION_TIMEOUT`<br>`--hono.coap.dtlsRetransmissionTimeout` | no | 2000 | The timeout in milliseconds for DTLS retransmissionsnumber. |
 | `HONO_COAP_DEFAULTS_ENABLED`<br>`--hono.coap.defaultsEnabled` | no | `true` | If set to `true` the protocol adapter uses *default values* registered for a device to augment messages published by the device with missing information like a content type. In particular, the protocol adapter adds default values registered for the device as (application) properties with the same name to the AMQP 1.0 messages it sends downstream to the AMQP Messaging Network. |
+| `HONO_COAP_EXCHANGE_LIFETIME`<br>`--hono.coap.exchangeLifetime` | no | 247000 | The exchange lifetime in milliseconds. According RFC 7252, that value is 247s. Such a large time requires also a huge amount of heap. That time includes a processing time of 100s and retransmissions of CON messages. Therefore a practical value could be much smaller.|
```

Comment:
is milliseconds the right unit here? My understanding is that the lifetime is defined in seconds rather than millis ...

Reply:
It's the unit used by californium since years. If other units are used here, I'm not sure, if that doesn't mix up users, which are also common to Californium. So I would keep the millis.

Reply:
Then let's stick to millis for the time being